docs/transactions.txt

```diff
@@ -60,21 +60,8 @@ This guide contains the following sections:
 Requirements and Limitations
 ----------------------------
 
-To perform transactions in MongoDB, you must use the following MongoDB
```

Comment:
Removing because this doesnt seem to be true
https://www.mongodb.com/docs/manual/core/transactions/#transactions-api

Reply:
Transactions are not supported on standalone servers.

Reply:
Its weird thats hidden away there instead of on the main transactions page! I'll get a fix in

Reply:
https://github.com/mongodb/laravel-mongodb/pull/3389